src/pattern-library/patterns/aside/aside.njk

```diff
@@ -1,3 +1,11 @@
-<div class="w-aside w-aside--{{ data.type if data.type else 'note' }}">
-  <p>{% if data.title %}<strong>{{ data.title }}</strong>: {% endif %}{{ data.content | safe }}</p>
+<div class="aside flow {{ data.utilities.main }}">
```

Comment:
@andy-piccalilli I wanted to double check that the `{% Aside %}` shortcode will still be supported as well? I'm realizing now that we didn't specifically discuss this, but some of the shortcodes are used a ton and the signatures are the same on both sites, so we'd want to keep them.

Some of the lesser used shortcodes could maybe go away, for instance I don't think `{% Banner %}` is really used that much, but Aside and Details would probably need to remain as shortcodes. Happy to discuss more if this throws a spanner into the works 😅

Reply:
After this phase of patterns being built, we're going to need to spend some time considering how this stuff will be integrated because ideally, all the existing shortcodes should support the new stuff because we don't want a massive content migration issue.

Each "page" that has the new CSS stuff will have `CSS_ORIGIN = 'next'`, so we might be able to lean into that?